starter-core/src/test/groovy/io/micronaut/starter/feature/awslambdacustomruntime/AwsLambdaCustomRuntimeSpec.groovy

```diff
@@ -64,7 +65,7 @@ class AwsLambdaCustomRuntimeSpec extends BeanContextSpec implements CommandOutpu
         when:
         def output = generate(
                 ApplicationType.FUNCTION,
-                new Options(language, TestFramework.JUNIT, BuildTool.MAVEN),
+                new Options(language, TestFramework.JUNIT, BuildTool.MAVEN, JdkVersion.JDK_11),
```

Comment:
11 is the default. why did you set explicitely Jdk 11 ? Just to be convey that it uses 11?

Reply:
JDK 11 is not the default when running the tests. The application gets the Java version from the JDK uses: https://github.com/micronaut-projects/micronaut-starter/blob/0a85901b0ed2d52836818e13ab17df9d1a444c78/starter-core/src/main/java/io/micronaut/starter/util/VersionInfo.java#L93-L92.
This makes that in Github Actions, when using JDK 14, all those tests related to graalvm fail.

Reply:
ok!